### PR TITLE
apns: Support for content-available inside aps payload

### LINKED ISSUE #28: apns: Does not support 'content-available'?
It seems that gaurun dose not support 'content-available' property on apns. Is there any reasons to do so? (or I misundestanding?)

I wrote a small patch to use 'content-available' on gaurun and I would like to send a PR.

### PR DESCRIPTION
**Pull request by tanatana**
fix #28

### CODE REVIEW

**Comment by cubicdaiya** [comment]
Hi, @tanatana 

Run `make fmt` and update [SPEC.md](https://github.com/mercari/gaurun/blob/master/SPEC.md#post-push), please.

**Comment by tanatana** [comment]
@cubicdaiya Should I squash the commits?

**Comment by cubicdaiya** [comment]
> Should I squash the commits?

You don't have to do it.

File: SPEC.md [hunk 1]
```diff
@@ -45,18 +46,19 @@ The following JSON is a request-body example.
...
-|extend          |string array|extensible partition                     |-       |                |
+|name             |type        |description                              |required|note            |
+|-----------------|------------|-----------------------------------------|--------|----------------|
+|token            |string array|device tokens                            |o       |                |
+|platform         |int         |platform(iOS,Android)                    |o       |1=iOS, 2=Android|
+|message          |string      |message for notification                 |o       |                |
+|badge            |int         |badge count                              |-       |only iOS        |
+|sound            |string      |sound type                               |-       |only iOS        |
+|expiry           |int         |expiration for notification              |-       |only iOS        |
+|content_available|bool        |indicate  that new content is available  |-       |only iOS        |
```
Comment (cubicdaiya) [inline SPEC.md]: Decrease space between `indcate` and `that`, please.

**Comment by cubicdaiya** [comment]
Hi @tanatana, 

I commented again. Please check it.

**Comment by tugusuke** [comment]
@cubicdaiya 
I'm sorry about my mistake. 
I removed an extra space.

**Comment by tanatana** [comment]
@cubicdaiya 
I'm sorry about my mistake. 
I removed an extra space.

**Comment by cubicdaiya** [comment]
Merged. Thanks.

**Pull request merged**